feature: Add Release note for PMD Legacy removal CY-5203 (#1047)
codacy/docs: docs/release-notes/cloud/cloud-2022-02-16-pmd-legacy-removal.md

---
rss_title: Codacy release notes RSS feed
rss_href: /feed_rss_created.xml
---

# Removal of PMD (Legacy) February 16, 2022

> On the week of February 16th 2022 we'll be removing the tool **PMD (Legacy)** from Codacy.

Reviewer: ```suggestion
On the week of February 16th 2022 Codacy will stop supporting the **PMD (Legacy)** linter.
```

Reviewer: dunno if linter is better than tool though, probably @prcr will be able to help here

Reviewer 2: Throughout the docs we consistently refer to "tools", for example:

- https://docs.codacy.com/getting-started/supported-languages-and-tools/
- https://docs.codacy.com/related-tools/local-analysis/client-side-tools/

Would you say that "linter" is more correct, or more specific perhaps? We can always consider adjusting the wording used on the docs, although I would do it as a separate task. :slightly_smiling_face:

The **PMD (Legacy)** tool runs [PMD 5.8.1](https://github.com/pmd/pmd/releases/tag/pmd_releases%2F5.8.1){: target="_blank"}, and was introduced to allow a smoother migration to PMD 6.

However, the legacy version of PMD is almost five years old and is no longer being maintained by the tool developers, while PMD 6 is now very stable and provides newer features.

## If you were using PMD (Legacy)

> To continue using PMD to analyze your repositories, [enable the **PMD** tool](../../repositories-configure/configuring-code-patterns.md) on the **Code patterns** page of your repositories.

Reviewer 2: Can we also include here a way to let customers reach Support?

```suggestion
To continue using PMD to analyze your repositories, [enable the **PMD** tool](../../repositories-configure/configuring-code-patterns.md) on the **Code patterns** page of your repositories.

If you have any questions or need help, please contact <mailto:support@codacy.com>.
```
